tests/waku_rln_relay/test_rln_group_manager_onchain.nim

```diff
@@ -285,6 +92,19 @@ suite "Onchain group manager":
     (await manager2.init()).isErrOr:
       raiseAssert "Expected error when contract address doesn't match"
 
+  # FAILED
```

Comment:
is this the test that's failing now?

Reply:
No, sorry. That was a cue for myself which I intended to delete.